excel.go

```diff
@@ -213,7 +206,8 @@ func updateMACFinUsers(f *excelize.File, input *Input) error {
 	rowOffset := input.RowOffset
 
 	for i, row := range rows[rowOffset:] {
-		if row == nil {
+		// check if row is empty or too short
+		if len(row) < max(userX, passwordX) {
```

Comment:
`<=`

Reply:
Yes. However, I decided to write a function `isValid` that checks a row. This can be used in the 2 places the code checks: `getMACFinUsers` and `updateMACFinUsers`. Instead of checking whether a row is too short, I check whether username or password are empty. This check covers short rows and long rows. (A long row is a row with a non-empty cell in a col > max(username, password). If row is long, with empty username or password, the code has undesirable behavior.